### PR TITLE
Restore basic_easy_shaped heart.get from 0.5 -> 15 (takes longer to get there)

### PR DESCRIPTION
**Pull request by relh**
This basic_easy_shaped now gets to 15 reward, but has a different slope to get there.

no longer stacked on richard-only-one-move

[Asana Task](https://app.asana.com/1/1209016784099267/project/1210348820405981/task/1211109232392214)

### CODE REVIEW

File: agent/src/metta/agent/pytorch/latent_attn_tiny.py [hunk 1]
```diff
@@ -143,12 +143,10 @@ def __init__(self, env, input_size=128, hidden_size=128):
...
 
-        # Action heads - will be initialized based on action space
-        action_nvec = self.action_space.nvec if hasattr(self.action_space, "nvec") else [100]
+        # Create action heads using mixin pattern
+        from metta.agent.pytorch.pytorch_agent_mixin import PyTorchAgentMixin
 
-        self.actor_heads = nn.ModuleList(
-            [pufferlib.pytorch.layer_init(nn.Linear(512 + 16, n), std=0.01) for n in action_nvec]
-        )
+        self.actor_heads = PyTorchAgentMixin.create_action_heads(self, env, input_size=512 + 16)
```
Comment (rjwalters) [inline agent/src/metta/agent/pytorch/latent_attn_tiny.py:149]: can we avoid magic numbers? where do 512 and 16 come from?

File: experiments/recipes/arena_basic_easy_shaped.py [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+        border_width=6,
+        instance_border_width=0,
+        root=metta.map.scenes.random.Random.factory(
+            params=metta.map.scenes.random.Random.Params(
+                agents=6,
+                objects={
+                    "mine_red": 10,
+                    "generator_red": 5,
+                    "altar": 5,
+                    "block": 20,  # CRITICAL: Blocks were in old config
```
Comment (rjwalters) [inline experiments/recipes/arena_basic_easy_shaped.py:58]: leftover LLM comments?

Comment (relh) [reply experiments/recipes/arena_basic_easy_shaped.py:58]: yeah I haven't cleaned it up yet

**Review by Al-does — APPROVED** [review]
